client/app/telecom/telephony/billingAccount/administration/addGroup/telecom-telephony-billing-account-administration-add-group.html

```diff
@@ -1,66 +1,56 @@
-                            data-ng-disabled="!AddGroupCtrl.contractsAccepted || AddGroupCtrl.ordering"
-                            data-ng-click="AddGroupCtrl.orderGroup()">
-                        <oui-spinner class="mr-2"
-                                     data-ng-if="AddGroupCtrl.ordering"
-                                     data-size="s">
-                        </oui-spinner>
-                        <span data-translate="telephony_add_group_order"></span>
-                        <span>({{ AddGroupCtrl.getDisplayedPrice() }})</span>
-                    </button>
-                </div>
+        <oui-message type="info" data-ng-if="!AddGroupCtrl.order">
+            <span data-translate="telephony_add_group_infos"></span>
+        </oui-message>
 
-                <div data-ng-if="AddGroupCtrl.order">
-                    <p>
-                        <i class="ovh-font ovh-font-success text-success h3 mb-0 mr-2"></i>
-                        <span data-translate="telephony_add_group_success"></span>
-                    </p>
-                    <p>
-                        <span data-translate="telephony_add_group_success_bc"></span>
-                        <a target="_blank"
-                           rel="noopener"
-                           data-ng-href="{{ AddGroupCtrl.order.url }}">
-                            <span data-translate="telephony_add_group_success_bc2"></span>
-                            <i class="fa fa-external-link"></i>
-                        </a>
-                        .
-                    </p>
-                </div>
+        <div data-ng-if="!AddGroupCtrl.order">
+            <p data-translate="telephony_add_group_contracts"></p>
+            <div data-contracts="AddGroupCtrl.contracts"
+                 data-full-text="true"
+                 data-contracts-validated="AddGroupCtrl.contractsAccepted">
             </div>
+
+            <button type="button"
+                    class="btn btn-primary"
+                    data-ng-disabled="!AddGroupCtrl.contractsAccepted || AddGroupCtrl.ordering"
+                    data-ng-click="AddGroupCtrl.orderGroup()">
+                    <span data-translate="telephony_add_group_order"></span>
+                <span>({{ AddGroupCtrl.getDisplayedPrice() }})</span>
+            </button>
+        </div>
+
+        <div data-ng-if="AddGroupCtrl.order">
+            <p>
+                <i class="ovh-font ovh-font-success text-success h3 mb-0 mr-2"></i>
+                <span data-translate="telephony_add_group_success"></span>
+            </p>
+            <p>
+                <span data-translate="telephony_add_group_success_bc"></span>
+                <a target="_blank"
+                   rel="noopener"
+                   data-ng-href="{{ AddGroupCtrl.order.url }}">
+                    <span data-translate="telephony_add_group_success_bc2"></span>
+                    <i class="fa fa-external-link"></i>
```

Comment:
We should depend only on ui-kit not font awesome
oui-icon-external_link

Reply:
👍